keras_nlp/src/utils/preset_utils.py

```diff
@@ -192,6 +200,43 @@ def get_file(preset, path):
         )
 
 
+def load_preset_from_gcs(fname, url, cache_subdir):
+    """Much of this is adapted from get_file of keras core."""
+    if url is None:
```

Comment:
kill this error, we should never be passing none here, and this error would not be user actionable at all (they aren't calling a function with a `url` argument.

Reply:
still need to do this i think?